### DISCUSSION TITLE
nbsim and nbcores flag usage

### DISCUSSION THREAD

**Discussion by Maryemt** [avalanche-org/Gen-Assoc#62] [Show and tell]
There is no need to use --nbsim and --nbcores flags when theoretical analysis is selected by the user. 
mtdt.r and run_analysis.r uses flag detection to know what kind of analysis is selected so having these flags can affect the results.

to call runanalysis in a theoretical case these are the flags to provide

runanalysis.r --pedfile  --mapfile  --phenfile --phen --jobtitle --gi  --marketset (if multimarker is selected if not dont use the flag)

**Comment by Maryemt** [comment]
#58 see this issue

**Comment by Maryemt** [comment]
To help you here are the flags to use to run runanalysis.r in different scenarios
User wants to do a:
- empirical single marker analysis : runanalysis.r --pedfile --mapfile --phenfile --phen --nbsim --nbcores --jobtitle --gi
- theoretical SM analysis: runanalysis.r --pedfile --mapfile --phenfile --phen --jobtitle --gi 
- empirical Multi-marker analysis: runanalysis.r --pedfile --mapfile --phenfile --phen --nbsim --nbcores --marketset --jobtitle --gi
-  theoretical MM analysis: runanalysis.r --pedfile --mapfile --phenfile --phen  --marketset --jobtitle --gi

To sum up : 
 --nbsim --nbcores flags are only used when an empirical run is performed 
--marketset only when multimarker is selected 

otherwise those flags are not used making their value = NULL when testing their presence in the command

YW :)

**Comment by Maryemt** [comment]
Here are the remarks i have while running tests 
empirical single marker analysis : nbsim and nbcores are not passed as arguments while writing the command to run runanalysis.r 
empirical Multi-marker analysis: same issue in this case
theoretical SM analysis: you assigned default values to nbsim (0) ad nbcores(1) and they are not to be used in theoretical cases
theoretical MM analysis: same issue
